### PR TITLE
0.9 updates

### LINKED ISSUE #107: Change test document URLs to stage
Many of the Whole Tale links point to the dev or production instance. Now that we're set on deploying to stage for RC, we should probably point them to stage.

### PR DESCRIPTION
**Pull request by ThomasThelen**
### Settings Page
This is a new page in the UI that needs docs. The docs provide info on what the page does and which accounts can be connected. Note that I've left the screenshot out for dataone. I'll add the screenshot when we have WT deployed with production settings so that we see production dataone instead of just the stage 2 environment.

### Publishing
Updated publishing docs to point the users to the settings page to connect their accounts if they haven't.

### Misc Notes (release info, features, etc)
Feel free to modify the changes I've made to these.

### Test Plan
I took the DataONE specifics out of the publishing tests and added tests for connecting to external accounts.

Should resolve #107, #116, and be part of #111

### CODE REVIEW

File: users_guide/settings.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. `Zenodo <https://zenodo.org>`_
+2. `Zenodo Sandbox <https://sandbox.zenodo.org>`_
+
+.. figure:: images/settings/connect_zenodo.png
+     :align: center
+
+     Connecting your account to Zenodo.
+
+.. _Zendo token page : https://zenodo.org/account/settings/applications/tokens/new/
```
Comment (bodom0015) [inline users_guide/settings.rst]: Very minor, but noticed a few typos in here: `Zendo` => `Zenodo`

File: ISSUE_TEMPLATE/test_plan.md [hunk 2]
```diff
@@ -393,38 +422,31 @@ The purpose of these tests are to confirm that the metadata files (manifest.json
...
       1. `environment.json`
-  1. Confirm that the `Data Set Creators` section lists your account
-  1. Confirm that the `Data Set Usage Rights` matches the license on the Tale's Run > metadata page
   1. Confirm that the `Published Location` on the Run > metadata page is correct
 
 * [ ] Case 2: Re-published Tale
 1. Launch a Tale that has been published before
 1. Take note of the `Published Location` in Run > metadata
-1. Re-publish the Tale to the DataONE Development repository
+1. Re-publish the Tale to the a repository
```
Comment (bodom0015) [inline ISSUE_TEMPLATE/test_plan.md]: Extraneous `the` here

File: users_guide/settings.rst (hunk 1 above)
Comment (ThomasThelen) [reply users_guide/settings.rst]: Fixed in d700f81

File: ISSUE_TEMPLATE/test_plan.md (hunk 2 above)
Comment (ThomasThelen) [reply ISSUE_TEMPLATE/test_plan.md]: Fixed. Thanks!

**Pull request merged**